.github/workflows/base.yml

```diff
@@ -0,0 +1,42 @@
+name: base
+
+on:
+  schedule:
+    - cron: '0 15 * * *'
+  pull_request:
+    paths:
+      - '.github/workflows/base.yml'
+
+jobs:
+  examples:
+
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [3.6, 3.7, 3.8, 3.9]
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: setup-python${{ matrix.python-version }}
+      uses: actions/setup-python@v2
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: Install (Python)
+      run: |
+        python -m pip install --upgrade pip
+        pip install --progress-bar off -U setuptools
+        pip install git+https://github.com/optuna/optuna.git
+        python -c 'import optuna'
+
+        pip install -r requirements.txt
+    - name: Run examples
+      run: |
+        python max_trials_callback.py
+        python enqueue_trial.py
+        python quadratic_simple.py
+        python simple_pruning.py
+        python skimage/skimage_lbp_simple.py
+        python -m py_compile rapids_simple.py  # due to difficulty importing, compile only
+        python -m py_compile ray/ray_joblib.py  # due to difficulty importing, compile only
```

Comment:
Maybe nits but the workflow files used to reflect the file structure of the examples. Should we stick to this convention with `skimage` and `ray` to keep it easy to find what is tested where?